developer-docs-site/docs/concepts/basics-txns-states.md

```diff
@@ -54,5 +54,5 @@ In the figure:
 | Accounts **A** and **B** | Represent Alice's and Bob's accounts on the Aptos Blockchain |
 | **S<sub>N-1</sub>** | Represents the (**N-1**)th state of the blockchain. In this state, Alice's account **A** has a balance of 110 Aptos Coins, and Bob's account **B** has a balance of 52 Aptos Coins. |
 | **T<sub>N</sub>** | This is the **N**th transaction executed on the blockchain. In this example, it represents Alice sending 10 Aptos Coins to Bob. |
-| **F** | It is a deterministic function. **F** always returns the same final state for a specific initial state and a specific transaction. If the current state of the blockchain is **S<sub>N-1</sub>**, and transaction **T<sub>N</sub>** is executed on state **S<sub>N-1</sub>**, the new state of the blockchain is always **S<sub>N</sub>**. The Aptos Blockchain uses the [Move language](https://aptos.github.io/move) to implement the deterministic execution function **F**. |
+| **F** | It is a deterministic function. **F** always returns the same final state for a specific initial state and a specific transaction. If the current state of the blockchain is **S<sub>N-1</sub>**, and transaction **T<sub>N</sub>** is executed on state **S<sub>N-1</sub>**, the new state of the blockchain is always **S<sub>N</sub>**. The Aptos Blockchain uses the Move language to implement the deterministic execution function **F**. |
```

Comment:
Good catch. Actually we can use this link: https://move-language.github.io/move/